MAUI/DataForm/DataForm-settings.md

```diff
@@ -0,0 +1,853 @@
+</td>
+</tr><tr>
+<td>
+DataFormAutoCompleteItem
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+<td>
+Generated for enum type property. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormPickerItem
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for enum type property. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormRadioGroupItem
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the enum type property.
+</td>
+</tr>
+</table>
+
+You can customize the property settings or cancel the generation of `DataFormItem` by handling the `GenerateDataFormItem` event.
+
+## Cancel the auto-generation data editor
+
+You can cancel the specific data editor by handling the `GenerateDataFormItem` event or by defining display attribute to avoid the particular data field being displayed.
+
+#### Using attributes
+
+You can set `AutoGenerateField` to `false` for canceling the data editor generation.
+
+{% tabs %}
+{% highlight C# %}
+
+[Display(AutoGenerateField = false)]
+public int ID { get; set; } 
+    
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+In the following code, the data editor generation for the `MiddleName` property is canceled by setting the `Cancel` property to true of the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem.FieldName == "MiddleName")
+    {
+        e.Cancel = true;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The `GenerateDataFormItem` event will be raised only when `AutoGenerateItems` is set to true.
+
+For `AutoGenerateItems` false you can use settings properties directly by using the `DataFormItem`. Please refer here to know more about creating data editors explicitly.
+
+## Change the auto-generated editor settings
+
+You can change the data editor settings by using the property of `DataFormItem` in the `GenerateDataFormItem` event when `AutoGenerateItems` is true.
+
+Here, `Salary` data field is restricted from being edited in the data form.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null)
+    {
+        if (e.DataFormItem.FieldName == "MiddleName")
+        {
+             e.DataFormItem.IsReadOnly = true;
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Change the autogenerated editor visibility
+
+You can change the specific editor visibility by using the `IsVisible` property in the `DataFormItem`.
+
+Here, `LastName` data field will be hidden.
```

Comment:
Here, the `LastName` data field will be hidden.

Reply:
updated